.github/workflows/master.yml

```
@@ -477,4 +478,4 @@ jobs:
       - name: Run telemetry tests
         run: |
```

```suggestion
        env:
          _MLFLOW_TESTING_TELEMETRY: true
        run: |
```

can we set `_MLFLOW_TESTING_TELEMETRY` here?